File: ejercicio.py
personas = [];
telefonos = [];
nombre = input("Ingrese su nombre")
apellido = input("Ingrese su apellido")
dni = int(input("Ingrese su dni"))

cantidadNumeros = int(input("¿Cuántos números de teléfono desea ingresar?"))
for i in range(cantidadNumeros):
    telefono = input("Ingrese su número de teléfono")
    telefonos.append(telefono)

personas.append(nombre)
personas.append(apellido)
personas.append(dni)
personas.append(telefonos)

print(personas)

File: ejercicio.js
const prompt = require('prompt-sync')();

let personas = [];
let telefonos = [];

let nombre = prompt("Ingrese su nombre: ");
let apellido = prompt("Ingrese su apellido: ");
let dni = parseInt(prompt("Ingrese su DNI: "));
let cantidadNumeros = parseInt(prompt("¿Cuántos números de teléfono desea ingresar? "));

for (let i = 0; i < cantidadNumeros; i++) {
    let telefono = prompt("Ingrese el número de teléfono : ");
    telefonos.push(telefono);
}

personas.push(nombre, apellido, dni, telefonos);
console.log(personas);
